images/README.md

```diff
@@ -3,12 +3,22 @@ Image on Docker Hub: https://hub.docker.com/r/fnproject/python
 ## Building Python images
 
 ```sh
-pushd build/3.7.1; docker build -t fnproject/python:3.7.1-dev .; popd
```

Comment:
this file should just be a script probably, but this is ok for now

Reply:
that's something in my TODO, there's a need to do automated releases as well that is not the case for now.

Reply:
https://github.com/fnproject/dockers/blob/master/python/build.rb